weaver/cli.py

```diff
@@ -626,13 +704,39 @@ def make_parser():
     add_url_param(op_describe)
     add_process_param(op_describe)
 
-    op_execute = ops_parsers.add_parser("execute", help="Submit a job execution for an existing process.")
+    op_execute = ops_parsers.add_parser(
+        "execute", formatter_class=InputsFormatter,
+        help="Submit a job execution for an existing process."
+    )
     add_url_param(op_execute)
     add_process_param(op_execute)
-    # FIXME: support cumulative inputs for convenience? (ex: '-I input=value -I array=1,2 -I other=http://file')
     op_execute.add_argument(
-        "-I", "--inputs", dest="inputs",
-        help="File path or URL reference to JSON or YAML contents defining job inputs with OGC-API or CWL schema."
+        "-I", "--inputs", dest="inputs", nargs="+",
```

Comment:
Using `nargs="+"` can result in some conflict with the `url` positional argument`.

```
weaver execute -p asv -I /home/cumminch/asv/asv_execute.json http://localhost:4001

usage: weaver execute [-h] [-p PROCESS_ID] [-I INPUTS [INPUTS ...]] [-M]
                      [-T TIMEOUT] [-W INTERVAL]
                      url
weaver execute: error: the following arguments are required: url
```

Reply:
Should be resolved now.